src/gamma_inc.jl

```diff
@@ -488,6 +733,111 @@ function gamma_inc_fsum(a::Float64, x::Float64)
     return (1.0 - q, q)
 
 end
+"""
+    gamma_inc_inv_psmall(a,p)
+
+Compute x0 - initial approximation when `p` is small.
+Here we invert the series in Eqn (2.20) in the paper and write the inversion problem as:
+```math
+x = r(1 + \\sum_{1}^{\\infty}a(-1)^{n}x^{n}/(a+n)n!)^{-1/a}
+```
+where ``r = (p\\Gamma(1+a))^{1/a}``
+Inverting this relation we obtain ``x = r + \\sum_{2}^{\\infty}c_{k}r^{k}``
+"""
+function gamma_inc_inv_psmall(a::Float64, p::Float64)
+    logr = (1.0/a)*(log(p) + logabsgamma(a + 1.0)[1])
+    r = exp(logr)
+    m=0
+    a² = a*a
+    a³ = a²*a
+    a⁴ = a³*a
+    ap1 = a + 1.0
+    ap1² = (ap1)*ap1
+    ap1³ = (ap1)*ap1²
+    ap1⁴ = ap1²*ap1²
+    ap2 = a+2.0
+    ap2² = ap2*ap2
+    ck1 = 1.0
+    ck2 = 1.0/(1.0+a)
+    ck3 = 0.5*(3*a+5)/(ap1²*(a+2))
+    ck4 = (1.0/3.0)*(@horner(a,31,33,8))/(ap1³*ap2*(a+3))
+    ck5 = (1.0/24.0)*(@horner(a, 2888, 5661, 3971, 1179, 125))/(ap1⁴*ap2²*(a+3)*(a+4))
+    x0 = @horner(r, 0.0, ck1, ck2, ck3, ck4, ck5)
+    return x0
+end   
+"""
+    gamma_inc_inv_qsmall(a,q)
+
+Compute x0 - initial approximation when `q` is small from ``e^{-x_{0}} x_{0}^{a} = q \\Gamma(a)``.
+Asymptotic expansions Eqn (2.29) in the paper is used here and higher approximations are obtained using 
+```math
+x \\sim x_{0} - L + b* \\sum_{1}^{\\infty} d_{k}/x_{0}^{k} 
+```
+where b = 1-a, L = ln(x0)
+""" 
+function gamma_inc_inv_qsmall(a::Float64, q::Float64)
+    m =0
+    b=1.0-a
+    b²=b*b
+    b³=b²*b
+    eta=sqrt(-2/a*log(q*gammax(a)*sqrt(2*pi)/sqrt(a)))
+    x0 = a*lambdaeta(eta)
+    l = log(x0)
+
+    if a > 0.12 || x0 > 5
+        l² = l*l
+        l³ = l²*l
+        l⁴ = l³*l
+        r = 1.0/x0
+        ck1 = l - 1.0
+        ck2 = (@horner(l, 3*b+2, -2*b-2, 1))/2.0       
+        ck3 = (@horner(l, -11*b²-24*b-12, 24*b+12+6*b², -6-9*b, 2))/6.0
```

Comment:
it is a little absurd, but you can do nested `@horner` calls here.

Reply:
sure!